ajax/libs/hammer-time/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "hammer-time",
+  "filename": "hammer-time.min.js",
+  "description": "A fast click based on the touch-action css property",
+  "version": "1.1.0",
+  "author": "Alex Schmitz",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/hammerjs/hammer-time.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "touch-action",
+    "touch",
+    "action",
+    "css",
+    "touchevents",
+    "events",
+    "pointerevents",
+    "pointer",
+    "click",
+    "delay",
+    "hammer"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/hammerjs/hammer-time.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "hammer-time*"
```

Comment:
I think you don't need to minify the file by yourself.
please add the `hammer-time.min.js` which under `dist` folder.
ref: https://github.com/hammerjs/hammer-time/blob/v1.1.0/dist/hammer-time.min.js